packages/ui/package.json

```diff
@@ -21,6 +21,7 @@
     "react-markdown": "^4.3.1",
     "react-scripts": "3.4.1",
     "react-syntax-highlighter": "^12.2.1",
+    "react-virtualized": "^9.22.2",
```

Comment:
> If you're considering adding react-virtualized to a project, take a look at react-window as a possible lighter-weight alternative.

says the author of both `react-virtualized` and `react-window`. Should we use `react-window` as well or do we need features only supported by `react-virtualized`?